src/main/java/mods/railcraft/world/level/block/CokeOvenBricksBlock.java

```diff
@@ -3,19 +3,35 @@
 import mods.railcraft.world.level.block.entity.multiblock.CokeOvenBlockEntity;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
+import net.minecraft.block.HorizontalBlock;
 import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.DirectionProperty;
+import net.minecraft.state.StateContainer;
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.util.ActionResultType;
+import net.minecraft.util.Direction;
 import net.minecraft.util.Hand;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.BlockRayTraceResult;
 import net.minecraft.world.IBlockReader;
 import net.minecraft.world.World;
 
 public class CokeOvenBricksBlock extends Block {
+  public static final BooleanProperty ISLIT = BooleanProperty.create("lit");
```

Comment:
Rename this field to `LIT` and use `BlockStateProperties.LIT` rather than creating a new property.